macros/ArticlesByTag.ejs

```diff
@@ -18,22 +18,26 @@ var l10n = {
         "en-US": "There have been no articles written yet. <a href=\"/en-US/docs/MDN/Getting_started\">Please consider contributing</a>.",
         "zh-CN": "这些文章尚未撰写。<a href=\"/zh-CN/docs/MDN/Getting_started\">欢迎贡献</a>。",
         "fr"   : "Aucun article n'a encore été écrit ou traduit. <a href=\"/fr/docs/MDN/Débuter_sur_MDN\">N'hésitez pas à contribuer.</a>",
-        "ru"   : "Вы ещё не написали статей. <a href=\"/ru/docs/MDN/Getting_started\">Пожалуйста, посмотрите о вкладе</a>."
+        "ru"   : "Вы ещё не написали статей. <a href=\"/ru/docs/MDN/Getting_started\">Пожалуйста, посмотрите о вкладе</a>.",
+        "pt-BR': "Não hà artigos escritos ainda. <a href=\"/pt-BR/docs/MDN/Primeiros_Passos\"> Por favor, considere contribuir </a>."
```

Comment:
Ok, I'll be reviewing all the submitted "Pull requests," and apologize for my inconvenience.